s2core/src/main/scala/com/kakao/s2graph/core/storage/redis/RedisRPC.scala

```diff
@@ -20,6 +21,9 @@ case class RedisGetRequest(key: Array[Byte], isIncludeDegree: Boolean = true) ex
    */
   lazy val degreeEdgeKey = Bytes.add(key, RedisRPC.DEGREE_EDGE_POSTFIX_BYTE)
 
+  // Todo: Not sure if limiting the number of properties is the right approach..
+  lazy val MaxPropNum = 100
```

Comment:
Why do you use limit number for vertex properties? If `zrangebylex` supports unlimit count as a parameter, we'd better use the unlimit count. Do you consider about fetch size because of rpc timeout?